src/styles/main.scss

```diff
@@ -152,6 +156,21 @@
   }
 }
 
+%centered-image {
```

Comment:
@shu this part is needed to let you adds an image to the center of a div. I use it in components